client/search-ui/src/components/CommitSearchResult.tsx

```diff
@@ -60,7 +60,7 @@ export const CommitSearchResult: React.FunctionComponent<Props> = ({
                     <VisuallyHidden>,</VisuallyHidden>
                 </Code>{' '}
                 <VisuallyHidden>Commited</VisuallyHidden>
-                <Timestamp date={result.committerDate} noAbout={true} strict={true} />
+                <Timestamp date={result.committerDate} noAbout={true} strict={true} utc={true} />
```

Comment:
suggestion: add a comment similar to this here explaining why this is in UTC.

```suggestion
                {/* Display commit date in UTC to match behavior of before/after filters */}
                <Timestamp date={result.committerDate} noAbout={true} strict={true} utc={true} />
```